NAMESPACE

```diff
@@ -3,15 +3,20 @@
 S3method(plot,safeTSim)
 S3method(print,safe2x2_result)
 S3method(print,safe2x2_test)
-S3method(print,safeTDesign)
-S3method(print,safeTResult)
+S3method(print,safeDesign)
 S3method(print,safeTSim)
-S3method(simulate,safeTDesign)
+S3method(print,safeTest)
+S3method(simulate,safeDesign)
 export(designFreqT)
+export(designFreqZ)
```

Comment:
Check

Reply:
Je heoft de namespace niet te bekijken. Dat laten we gewoon aan roxygen over.